includes/abstracts/calendar.php

```diff
@@ -570,11 +570,17 @@ public function set_time_format( $format = '' ) {
 	 */
 	public function set_datetime_separator( $separator = '' ) {
 
+		$separator_spacing = get_post_meta( $this->id, '_calendar_datetime_separator_spacing', true );
```

Comment:
Since this line is right on top of an empty check for `$seperator` I thought it was defining `$seperator`.

Can we group this new addition with the other related additions that start on line 578 (just above where we check `empty( $seperator_spacing )`?